### ISSUE TITLE
Why doesn't the gem come with all uploader validations?

### ISSUE THREAD

**Issue by Silex** [carrierwaveuploader/carrierwave#1858]
Hello,

https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/content_type_whitelist.rb and https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/uploader/file_size.rb are not present in the latest gem:

``` shell
root@23c2cd90c63f:/srv/app# ls -alh /usr/local/bundle/gems/carrierwave-0.10.0/lib/carrierwave/uploader
total 84K
drwxr-sr-x  2 root staff 4.0K Jan 22 15:16 .
drwxr-sr-x 11 root staff 4.0K Jan 22 15:16 ..
-rw-r--r--  1 root staff 5.6K Jan 22 15:16 cache.rb
-rw-r--r--  1 root staff  975 Jan 22 15:16 callbacks.rb
-rw-r--r--  1 root staff 5.5K Jan 22 15:16 configuration.rb
-rw-r--r--  1 root staff  364 Jan 22 15:16 default_url.rb
-rw-r--r--  1 root staff 2.7K Jan 22 15:16 download.rb
-rw-r--r--  1 root staff 1.5K Jan 22 15:16 extension_blacklist.rb
-rw-r--r--  1 root staff 1.6K Jan 22 15:16 extension_whitelist.rb
-rw-r--r--  1 root staff 1.2K Jan 22 15:16 mountable.rb
-rw-r--r--  1 root staff 2.5K Jan 22 15:16 processing.rb
-rw-r--r--  1 root staff 1.8K Jan 22 15:16 proxy.rb
-rw-r--r--  1 root staff  422 Jan 22 15:16 remove.rb
-rw-r--r--  1 root staff  703 Jan 22 15:16 serialization.rb
-rw-r--r--  1 root staff 3.0K Jan 22 15:16 store.rb
-rw-r--r--  1 root staff 1.1K Jan 22 15:16 url.rb
-rw-r--r--  1 root staff 9.4K Jan 22 15:16 versions.rb
```

Why that? Wouldn't `size_range` make https://github.com/carrierwaveuploader/carrierwave/wiki/how-to:-validate-attachment-file-size easier?

Also, the README mentions `def content_type_whitelist` which simply cannot work atm because `content_type_whitelist.rb` is not present.

**Comment by thomasfedb** [comment]
The master branch on GitHub is unreleased, and is targeting v1.0.0. The latest released version is v0.10.0. 

This is noted towards the top of the README in large bold font. Switch to the 0.10-stable branch for the relevant documentation. 

You can also use the master branch in your application by providing the Git repo in your Gemfile.

**Comment by Silex** [comment]
Alright, thanks.
